dynomitemanager/src/main/java/com/netflix/dynomitemanager/defaultimpl/DynomitemanagerConfiguration.java

```diff
@@ -704,11 +713,23 @@ public String getCommaSeparatedCassandraHostNames() {
     public boolean isEurekaHostSupplierEnabled() {
 	return configSource.get(CONFIG_IS_EUREKA_HOST_SUPPLIER_ENABLED, DEFAULT_IS_EUREKA_HOST_SUPPLIER_ENABLED);
     }
-    
+
     // Redis compatibility
     @Override
     public String getRedisCompatibleEngine() {
 	return configSource.get(CONFIG_DYNO_REDIS_COMPATIBLE_ENGINE, DEFAULT_REDIS_COMPATIBLE_ENGINE);
     }
-    
+
+    // Redis
+    // =====
+
+    /**
+    * Get the full path to the redis.conf configuration file.
+    * @return the {@link String} full path to the redis.conf configuration file
+    */
+    @Override
+    public String getRedisConf() {
```

Comment:
`IConfigSource` is not binded in the listener, so this may or may not work.

 Another approach is that we are already having `StorageProxy` inside the `DynomiteManagerConfiguration`. Currently, we do not use it. You can use that in implementing a `getStorageConfig` that calls an abstraction inside the `StorageProxy`.